pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -9,10 +9,22 @@
 , lib
 , stdenvNoCC
 , bintools ? null, libc ? null, coreutils ? null, shell ? stdenvNoCC.shell, gnugrep ? null
+, netbsd ? null, netbsdCross ? null
+, sharedLibraryLoader ?
+  if libc == null then
+    null
+  else if stdenvNoCC.targetPlatform.isNetBSD then
+    if libc != targetPackages.netbsdCross.headers then
```

Comment:
This should probably be written in a way that is able to handle the default case of `targetPackages` which is an empty attribute set.

Reply:
You mean in the non-cross case?  In that case, we don't have anything to set it to, because we don't have a native bootstrap yet.  So it wouldn't work either way.  Do you still think this should be changed even so?